### ISSUE TITLE
Maybe error in example for md-gutter

### ISSUE THREAD

**Issue by dima74** [vuematerial/vue-material#1111]
### Steps to reproduce
There is example of `md-gutter` in [layout docs](http://vuematerial.io/#/ui-elements/layout):

```
<md-layout md-gutter>
  <md-layout>
    <span>Automatic</span>
  </md-layout>

  <md-layout>
    <span>Automatic</span>
  </md-layout>
</md-layout>
```
[Link to source code for this example](https://github.com/vuematerial/vue-material/blob/master/docs/src/pages/ui-elements/Layout.vue#L649-L657)

But this doesn't work: [codepen](https://codepen.io/dima74/pen/OONRKG).
However in [docs code](https://github.com/vuematerial/vue-material/blob/master/docs/src/pages/ui-elements/Layout.vue#L551-L560) there is another code, with

`<md-layout :md-gutter="true" class="color">`

instead of

`<md-layout md-gutter>`.

I think you should change example code (that is change [this line](https://github.com/vuematerial/vue-material/blob/master/docs/src/pages/ui-elements/Layout.vue#L649))

### Which browser?

Chrome 61

### What is expected?
Some space between `md-layout` children. All works as expected in [this codepen](https://codepen.io/dima74/pen/YEqpXO) (there is code `<md-layout :md-gutter="true">` in this codepen)

### What is actually happening?

No space between `md-layout` children. Actual result can be seen in [this codepen](https://codepen.io/dima74/pen/OONRKG) (there is code `<md-layout md-gutter>` in this codepen)

Thanks for Vue Material!

**Comment by VdustR** [comment]
As this [codepen](https://codepen.io/VdustR/pen/LOZLEM):

I declared three components with `test` props for three cases of props type and pass the props without value in template:

> `Boolean`, `String`, `[String, Number, Boolean]`

While the prop type was `Boolean`, we got `true`;
while the prop type was `String`, we got `''`;
while the prop type was `[String, Number, Boolean]`, we got `''`;

`mdGutter`'s type is `[String, Number, Boolean]`, so the value would be `''` if you don't assign value to it in template.

I also tried to reorder the type array as `[Boolean, String, Number]`, still got `''`.

I think we should fix the documentation too.
